### ISSUE TITLE
Unsure of correct dimensions for cutting

### ISSUE THREAD

**Issue by fishe160** [fredvomjupiter/WordClock-NeoMatrix8x8#4]
Hi,
I'm new to laser cutting, trying to use the files you provided.
When I upload them to Ponoko, only the dimensions on the clear part of the enclosure passes the laugh test. The files for the letters and the pixel guard open as 1.2mm or 296.3mm square, which must be incorrect given the size of the neopixel matrix.
Is there something I have done wrong to corrupt this measurement? I am not familiar with .svg or .dxf files.
Thanks for the help, I've already got the electronics wired and working, just need this case to put them in!
matt
![Capture](https://user-images.githubusercontent.com/6764015/69109835-deca2680-0a46-11ea-89ce-3c7f662b021b.PNG)

**Comment by danowall** [comment]
The faceplate should be 83.82mm (or 3.3in) on a side. 
Some new AI and DXF versions of the files attached, hopefully these work better :)
[Wordclock 2D files.zip](https://github.com/andydoro/WordClock-NeoMatrix8x8/files/3923890/Wordclock.2D.files.zip)

**Comment by fishe160** [comment]
Thank you for the new files!
I meant to come back and update this sooner: the issue was Ponokos website. I got them to confirm via email that it was not correctly importing .svg files. It also tried to charge me $400 for them lol.
I finished putting it together tonight, very cool! Thanks for the directions, and the code, I learned a lot from dissecting it. 
Also your other projects on your website are pretty cool too.
Thanks!
matt
